### PR TITLE
fix IE11 Script Access Denied error.

### PR DESCRIPTION
**Pull request by doctyper**
This pull request fixes an issue that surfaced when we used core-js on a page containing iframes in IE11. Attempting to access properties on the iframe threw `Script Access Denied` errors.

### CODE REVIEW

File: modules/_classof.js [hunk 1]
```diff
@@ -5,12 +5,32 @@ var cof = require('./_cof')
...
+  }
+
+  if (it === null) {
+    return 'Null';
+  }
+
+  var O = Object(it);
+
+  // @@toStringTag case
+  if (TAG in O && typeof O[TAG] === 'string') {
```
Comment (doctyper) [inline modules/_classof.js]: The fix here was to add the `TAG in O` check.

**Comment by zloirock** [comment]
Thanks for the PR! Can you show a reproducible example for clarifying the problem? Some fixes for IE11 `Script Access Denied` already added.

**Comment by zloirock** [comment]
@doctyper so what about a reproducible example?

**Comment by doctyper** [comment]
@zloirock It's very difficult to abstract this into a reproducible example.

We use a React stack using Babel to transpile our code. Babel uses core-js to shim functionality, and that's where the bug surfaces for us.

I've tried to reproduce the error using just core-js but the fact that the bug only occurs on IE11/IE Edge makes this a very tedious process.

I do believe this is in some way similar to the fallbacks you added (I used them to track down this bug as well).

**Comment by zloirock** [comment]
The main problem here - `({get [Symbol.toStringTag](){ throw 42 }}) + ''` should throw error by the spec, but with `try / catch` it will not work. And performance issue. I also can't reproduce it. Need to think what we can do.

**Comment by zloirock** [comment]
Still can't reproduce it, but OK, added this fallback.

**Pull request closed without merging**